addons/project_issue/models/account_analytic_account.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from openerp import api, fields, models
+from openerp.exceptions import UserError
+from openerp.tools.translate import _
```

Comment:
Not required a separate import statement. Use: from openerp import api, fields, models, _ instead.